CMakeLists.txt

```diff
@@ -12,6 +12,8 @@ set(CMAKE_AUTOMOC ON)
 set(CMAKE_AUTOUIC ON)
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
The c++ standard is driven by lxqt-build-tools LXQTCompilerSettings.cmake module. Now I see, it was bumped to v17 a while ago. Please, remove this line.